cloudslang-compiler/src/test/resources/task-args-one-liner/flow_1.yaml

```diff
@@ -18,7 +18,7 @@ flow:
   workflow:
     - CheckWeather:
         do:
-          ops.check_Weather: city = city_name, country = str('Wonderland')
+          ops.check_Weather: city = ${ city_name }, country = ${ 'Wonderland' }
```

Comment:
I think that Wonderland should remain a string to keep the original functionality